fvm/blueprints/epochs.go

```diff
@@ -28,7 +28,7 @@ transaction {
 const deployEpochTransactionTemplate = `
 import FlowClusterQC from 0x%s
 
-transaction(clusterWeights: [{String: UInt64}]) {
+transaction(clusterWeights: [{Name: UInt64}]) {
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig